src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,139 @@
+package duke.tasks;
+
+import duke.exceptions.InvalidTaskDescriptionException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class TaskList {
+    ArrayList<Task> tasks;
+    int size;
+
+
+
+    public TaskList(int size) {
+        this.size = size;
+        this.tasks = new ArrayList<>(size);
+    }
+
+    //getter for arraylist
+
+    public ArrayList<Task> getTasks() {
+        return this.tasks;
+    }
+
+
+
+
```

Comment:
Consider removing redundant spaces in between codes to improve the code quality 😄